src/attrnotegroup.js

```diff
@@ -0,0 +1,109 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// Author Taehoon Moon 2016
+//
+// ## Description
+//
+// This file implements `AttrNoteGroup` which is used to format and
+// render attr notes; ClefNote, TimeSigNote and BarNote.
+
+import { Vex } from './vex';
+import { Flow } from './tables';
+import { Modifier } from './modifier';
+import { Formatter } from './formatter';
+import { Voice } from './voice';
+import { GraceNoteGroup } from './gracenotegroup';
```

Comment:
Oh, it's obviously, my mistake :) I'll update.

Firstly, I tested using GraceNoteGroup which works similarly (it also convert Note to Modifier).
And after I finished implementing, I had to remove that but forgot.